python/even_fibonacci_numbers.py

```diff
@@ -0,0 +1,17 @@
+"""
+Each new term in the Fibonacci sequence is generated by adding the previous two terms. By starting with 1 and 2, the first 10 terms will be:
+
+1, 2, 3, 5, 8, 13, 21, 34, 55, 89, ...
+
+By considering the terms in the Fibonacci sequence whose values do not exceed four million, find the sum of the even-valued terms.
+"""
+
+i = 1
+j = 2
+result = [i]
+
+while j < 4000000:
+    result.append(j)
+    i, j = j, j + i
```

Comment:
If possible even try to skip the iterations for odd number. this will further reduce the time complexity.